### PR TITLE
clearer instructions when using workers subdomain

### PR DESCRIPTION
**Pull request by stephanwesten**
Note that the command `wrangler preview --watch` directed me to example.com. So I got the impression that changes made to the index.html were not reflected. At least i did not see them initially. After making a few more changes, i did see the changes, but still it shows example.com in the address bar which is pretty confusing.

### CODE REVIEW

File: products/workers/src/content/platform/sites/start-from-scratch.md [hunk 1]
```diff
@@ -29,9 +29,15 @@ To start from scratch to create a Workers Site, follow these steps:
   $ wrangler preview --watch
   ```
 
-5. Decide where you’d like to publish your site to: [a workers.dev subdomain](/get-started/guide#configure-for-deploying-to-workersdev) or your [personal domain](/get-started/guide#optional-configure-for-deploying-to-a-registered-domain) registered with Cloudflare.
+5. Decide where you’d like to publish your site to: [a workers.dev subdomain](/get-started/guide#configure-for-deploying-to-workersdev) or your [personal domain](/get-started/guide#optional-configure-for-deploying-to-a-registered-domain) registered with Cloudflare. Update your `wrangler.toml`:
```
Comment (deadlypants1973) [inline products/workers/src/content/platform/sites/start-from-scratch.md]:
```suggestion
5. Choose whether you would like to publish your site to: [a `*.workers.dev` subdomain](/get-started/guide#configure-for-deploying-to-workersdev) or your [personal domain](/get-started/guide#optional-configure-for-deploying-to-a-registered-domain) registered with Cloudflare. Then, update your `wrangler.toml` file accordingly:
```

File: products/workers/src/content/platform/sites/start-from-scratch.md [hunk 2]
```diff
@@ -29,9 +29,15 @@ To start from scratch to create a Workers Site, follow these steps:
...
 
-  Then, update your `wrangler.toml`:
+  **workers.dev**: Make sure `workers_dev` is set to true. This is the default.
+Configure route to match your workers.dev subdomain: 
+  ```toml
+  route = "your-workers-subdomain.workers.dev/*"
+  ```
+
```
Comment (deadlypants1973) [inline products/workers/src/content/platform/sites/start-from-scratch.md]:
```suggestion
Configure `route` to match your `*.workers.dev` subdomain: 

  ```toml
  route = "your-workers-subdomain.workers.dev/*"
  ```

```

File: products/workers/src/content/platform/sites/start-from-scratch.md [hunk 3]
```diff
@@ -29,9 +29,15 @@ To start from scratch to create a Workers Site, follow these steps:
   $ wrangler preview --watch
   ```
 
-5. Decide where you’d like to publish your site to: [a workers.dev subdomain](/get-started/guide#configure-for-deploying-to-workersdev) or your [personal domain](/get-started/guide#optional-configure-for-deploying-to-a-registered-domain) registered with Cloudflare.
+5. Decide where you’d like to publish your site to: [a workers.dev subdomain](/get-started/guide#configure-for-deploying-to-workersdev) or your [personal domain](/get-started/guide#optional-configure-for-deploying-to-a-registered-domain) registered with Cloudflare. Update your `wrangler.toml`:
 
-  Then, update your `wrangler.toml`:
+  **workers.dev**: Make sure `workers_dev` is set to true. This is the default.
```
Comment (deadlypants1973) [inline products/workers/src/content/platform/sites/start-from-scratch.md]:
```suggestion
  **`*.workers.dev`**: Make sure `workers_dev` is set to true. This is the default.
```

**Review by deadlypants1973 — COMMENTED** [review]
content revision

File: products/workers/src/content/platform/sites/start-from-scratch.md [hunk 4]
```diff
@@ -29,9 +29,16 @@ To start from scratch to create a Workers Site, follow these steps:
   $ wrangler preview --watch
   ```
 
-5. Decide where you’d like to publish your site to: [a workers.dev subdomain](/get-started/guide#configure-for-deploying-to-workersdev) or your [personal domain](/get-started/guide#optional-configure-for-deploying-to-a-registered-domain) registered with Cloudflare.
+5. Choose whether you would like to publish your site to: [a `*.workers.dev` subdomain](/get-started/guide#configure-for-deploying-to-workersdev) or your [personal domain](/get-started/guide#optional-configure-for-deploying-to-a-registered-domain) registered with Cloudflare. Then, update your `wrangler.toml` file accordingly:
```
Comment (lukeed) [inline products/workers/src/content/platform/sites/start-from-scratch.md]: While this _could be_ said for the original text, the new text makes it sound like you can **only** pick one or the other.

When `workers_dev` is true, then it deploys to the subdomain & all routes are assumed. I'm fairly certain it cannot be modified and/or narrowed down to explicit paths on the subdomain.

You must define a `zone_id` and `route` (or `routes`) combination in order to deploy to a custom zone.

File: products/workers/src/content/platform/sites/start-from-scratch.md [hunk 5]
```diff
@@ -29,9 +29,16 @@ To start from scratch to create a Workers Site, follow these steps:
...
 
-  Then, update your `wrangler.toml`:
+  **`*.workers.dev`**: Make sure `workers_dev` is set to true. This is the default.
```
Comment (lukeed) [inline products/workers/src/content/platform/sites/start-from-scratch.md]: It is not on by default: https://developers.cloudflare.com/workers/cli-wrangler/configuration#keys

```suggestion
  **`*.workers.dev`**: Enable the `workers_dev` configuration.
```

File: products/workers/src/content/platform/sites/start-from-scratch.md [hunk 6]
```diff
@@ -29,9 +29,16 @@ To start from scratch to create a Workers Site, follow these steps:
...
+Configure `route` to match your `*.workers.dev` subdomain: 
+
+  ```toml
+  route = "your-workers-subdomain.workers.dev/*"
```
Comment (lukeed) [inline products/workers/src/content/platform/sites/start-from-scratch.md]:
```suggestion
  workers_dev = true
```

File: products/workers/src/content/platform/sites/start-from-scratch.md [hunk 7]
```diff
@@ -29,9 +29,16 @@ To start from scratch to create a Workers Site, follow these steps:
...
+  ```
+  
+  Or
```
Comment (lukeed) [inline products/workers/src/content/platform/sites/start-from-scratch.md]: lol – what to do here?

```suggestion
  And/Or
```

**Review by lukeed — CHANGES_REQUESTED** [review]

**Review by lukeed — APPROVED** [review]

File: products/workers/src/content/platform/sites/start-from-scratch.md [hunk 8]
```diff
@@ -29,28 +29,34 @@ To start from scratch to create a Workers Site, follow these steps:
...
 
   **Personal Domain**: Add your `zone_id` and a `route`.
 
   ```toml
   zone_id = "42ef.."
-  route = "example.com/*"
+  route = "https://example.com/*"
   ```
 
   <Aside>
```
Comment (deadlypants1973) [inline products/workers/src/content/platform/sites/start-from-scratch.md]:
```suggestion
  <Aside type="note">
```

File: products/workers/src/content/platform/sites/start-from-scratch.md [hunk 9]
```diff
@@ -29,28 +29,34 @@ To start from scratch to create a Workers Site, follow these steps:
...
 
-  __Note:__ Check out documentation on [Routes](/platform/routes) to configure `route` properly
+  __Note:__ Check out documentation on [Routes](/platform/routes) to configure `route` properly.
```
Comment (deadlypants1973) [inline products/workers/src/content/platform/sites/start-from-scratch.md]:
```suggestion
Refer to the documentation on [Routes](/platform/routes) to configure `route` properly.
```

**Review by deadlypants1973 — COMMENTED** [review]
content review

**Review by deadlypants1973 — APPROVED** [review]

File: products/workers/src/content/platform/sites/start-from-scratch.md [hunk 10]
```diff
@@ -29,28 +29,35 @@ To start from scratch to create a Workers Site, follow these steps:
...
   zone_id = "42ef.."
-  route = "example.com/*"
+  route = "https://example.com/*"
   ```
 
-  <Aside>
+  <Aside type="note">
 
-  __Note:__ Check out documentation on [Routes](/platform/routes) to configure `route` properly
+    Refer to the documentation on [Routes](/platform/routes) to configure `route` properly.
```
Comment (deadlypants1973) [inline products/workers/src/content/platform/sites/start-from-scratch.md:51]:
```suggestion
Refer to the documentation on [Routes](/platform/routes) to configure `route` properly.
```

**Comment by lukeed** [comment]
@deadlypants1973 The markdown parsing here is unhappy. I know spacing looks off, but this is the (currently) the build error, which signifies that it thinks the list item is closing sooner than it should be:

Screen Shot 2021-09-14 at 9 47 04 AM

**Comment by lukeed** [comment]
This PR is now building correctly. The failing CI has to do with "Publish preview" failing, which is failing because external forks are not given access to the base repo's CI secrets.

Thank you!

**Pull request merged**